triple-store/triple-store-api/src/main/java/com/powsybl/triplestore/api/PropertyBag.java

```diff
@@ -195,10 +197,19 @@ public boolean isClassProperty(String name) {
         return classPropertyNames.contains(name);
     }
 
+    public void setMultivaluedProperty(List<String> multiValuedPropertyNames) {
+        this.multiValuedPropertyNames = Objects.requireNonNull(multiValuedPropertyNames);
```

Comment:
set this as a copy to be protected against unsafe changes:
```java
this.multiValuedPropertyNames = new ArrayList<>(Objects.requireNonNull(multiValuePropertyNames));
```

Reply:
Done